<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorPemiWidget</class>
 <widget class="QWidget" name="GeneratorPemiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>610</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Генератор ПЭМИН</string>
  </property>
  <widget class="QPushButton" name="m_clear_pb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="m_editTime_dte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_timeSet_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_status_l">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Статус:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_back_pb">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../path.qrc">
     <normaloff>:/image/button_back.png</normaloff>:/image/button_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeComplect">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время на комплект</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_start_pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>СТАРТ</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPreliminarilyTime">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Предварительное время</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="m_loadSets_tw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>190</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">Каталог</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="m_previewTime_le">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_boxAntenneLow1GGz_cb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAntenneLow_1GGz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>До 1 ГГц</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_boxAntenneHeigh1GGz_cb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAntenneHeigh_1GGz">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>От 1 ГГц</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_antenneLow1GGz_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_antenneHeigh1GGz_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_multiplisity_ckb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>580</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Кратность 0,5</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_boxInjener_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_injener_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_boxTehnic_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_tehnic_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelInjener">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Инженер</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTehnic">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Техник</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_boxWorkplace_cb">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_workplace_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlochadka">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Площадка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_createTable_pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Создание s/n, №, ZNP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_saveSets_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_amplituda_sb">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ampl_res">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Размах:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_amplituda">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPreliminarilyTime_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранение</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_numSets_le">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_statusNumSets_l">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Комплект(ов)</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../path.qrc"/>
 </resources>
 <connections/>
</ui>
